deploy/monitoring/02-prometheus.yaml

```diff
@@ -98,30 +132,25 @@ data:
       - job_name: 'kube-state-metrics'
         static_configs:
           - targets: ['kube-state-metrics.kube-system.svc.cluster.local:8080']
-
       - job_name: 'ngsa-memory'
         static_configs:
-          - targets: [ 'ngsa-memory.default.svc.cluster.local:8080' ]
-
+          - targets: [ 'ngsa-memory.ngsa.svc.cluster.local:8080' ]
       - job_name: 'ngsa-cosmos'
         static_configs:
-          - targets: [ 'ngsa-cosmos.default.svc.cluster.local:8080' ]
-
+          - targets: [ 'ngsa-cosmos.ngsa.svc.cluster.local:8080' ]
```

Comment:
Same as others, I'd just use ngsa-memory and thus remove things that are targeting cosmos or java.